app/safe/internal/server/route/delete.go

```diff
@@ -30,7 +32,7 @@ func isSentinel(j audit.JournalEntry, cid string, w http.ResponseWriter, spiffei
 		return true
 	}
 
-	j.Event = audit.EventBadSvid
+	j.Event = event.BadSpiffeId
```

Comment:
The number of these event constants were increasing; and also we started using some common logging utilities; so I felt it’s time to move even-related constants to their own package too.